src/server/console/logind_console_services.cpp

```diff
@@ -78,9 +81,9 @@ class GErrorPtr
     ::boost::throw_line((int)__LINE__))
 
 std::unique_ptr<LogindSession, decltype(&g_object_unref)> simple_proxy_on_system_bus(
-    mir::GLibMainLoop& ml,
-    GDBusConnection* connection,
-    char const* object_path)
+    mir::GLibMainLoop &ml,
+    GDBusConnection *connection,
+    char const *object_path)
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style